hadoop-hdds/client/src/test/java/org/apache/hadoop/hdds/scm/storage/TestBlockInputStream.java

```diff
@@ -68,7 +72,6 @@
 /**
  * Tests for {@link BlockInputStream}'s functionality.
  */
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
Just curious, why was this removed?

Reply:
The `MockitoJUnitRunner` doesn't work with `ParameterizedTest`. So, I removed that declaration to let JUnit figure out the right runner itself. A little side effect is that we have to manually mock objects instead of just declaring mocks using `@Mock`.